service/src/main/java/com/codeforcommunity/requester/S3Requester.java

```diff
@@ -273,4 +275,31 @@ public static String uploadHTML(
 
     return String.format("%s/%s/%s", externs.getBucketPublicUrl(), directoryName, name);
   }
+
+  /**
+   * Delete the existing HTML file with the given name from the user uploads S3 bucket.
+   *
+   * @param name the name of the HTML file in S3 to be deleted.
+   * @param directoryName the directory of the file in S3 (without leading or trailing '/').
+   * @return previously existing HTML file URL if the deletion was successful.
+   * @throws InvalidURLException if the file does not exist.
+   * @throws SdkClientException if the deletion from S3 failed.
+   */
+  public static String deleteHTML(String name, String directoryName) {
+    String HTMLPath = directoryName + "/" + name;
+
+    // if the object does not exist, it throws a 403 for metadata?
+    try {
+      Boolean htmlExists = externs.getS3Client().doesObjectExist(externs.getBucketPublic(), HTMLPath);
+    } catch(AmazonServiceException e) {
+      throw new InvalidURLException();
+    }
+
+    // Create the request to delete the HTML
+    DeleteObjectRequest awsRequest = new DeleteObjectRequest(externs.getBucketPublic(), HTMLPath);
+
+    externs.getS3Client().deleteObject(awsRequest);
+
+    return String.format("%s/%s/%s", externs.getBucketPublicUrl(), directoryName, name);
```

Comment:
⛏️ Do we use the return value anywhere? If not, I think we can just remove it, and make the method void instead.